packages/scaffold-ui/src/partials/w3m-input-address/index.ts

```diff
@@ -97,7 +97,7 @@ ${this.value ?? ''}</textarea
 
   private async toggleInstructionFocus(focus: boolean) {
     if (this.instructionElementRef.value) {
-      await this.instructionElementRef.value.animate(
+      await this.instructionElementRef.value.animate?.(
```

Comment:
What is the use case if animate is not valid? Like almost all browsers should support this function, is there any exception? 

+ should we check for `instructionElementRef?.value?.animate` as well for the case ref is not beeing set yet?

Reply:
I added this to bypass a test issue, but will mock the ref so it's better